File: Android/app/src/main/java/com/dev334/trace/model/Attendance.java
package com.dev334.trace.model;

import com.dev334.trace.util.app.AppConfig;

import java.util.Date;
import java.util.List;

public class Attendance {
    private String date;
    private String department, semester, subject, token;
    private List<String> pictures;

    public Attendance(String date, String department, String semester, String subject,
                      List<String> urls, String token) {
        this.date = date;
        this.department = department;
        this.semester = semester;
        this.subject = subject;
        this.pictures = urls;
        this.token = token;
    }
}

File: Backend/model/attendance.js
const mongoose= require('mongoose')

const attendanceSchema = new mongoose.Schema({
    date: {
        type: String,
        required: true
    },
    department: {
        type: String,
        required: true
    },
    semester: {
        type: String,
        required: true
    },
    subject: {
        type: String,
        required: true
    },
    pictures: {
        type: [String],
        required: true
    }
})

const Attendance= mongoose.model('Attendance', attendanceSchema);
module.exports=Attendance;